packages/webcrypto-ed25519-polyfill/src/secrets.ts

```diff
@@ -138,3 +138,25 @@ export function verifyPolyfill(key: CryptoKey, signature: BufferSource, data: Bu
         return false;
     }
 }
+
+export function importKeyPolyfill(
+  format: KeyFormat,
+  keyData: BufferSource,
+  extractable: boolean,
+  keyUsages: readonly KeyUsage[]
+): CryptoKeyPair {
+  const bytes = bufferSourceToUint8Array(keyData);
+
+  if (format !== "raw") {
```

Comment:
If my memory serves me correctly, I _think_ that `raw` _is_ permitted for public keys (ie. verify usage) but not for private keys (ie. sign usage) which must come in PKCS#8 format.